spec/models/like_spec.rb

```diff
@@ -15,10 +15,10 @@
   subject do
     described_class.new(
       author: user,
-      post:
+      post: post
     )
   end
   it 'is valid with valid attributes' do
     expect(subject).to be_valid
   end
```

Comment:
- [ ] As requested by the previous reviewer, kindly add a test for the likes_counter, put in mind the scenario where a like is not yet added, and after adding a like.